bundles/org.openhab.core.io.transport.mqtt/src/main/java/org/eclipse/smarthome/io/transport/mqtt/sslcontext/SSLContextProvider.java

```diff
@@ -24,6 +24,7 @@
  *
  * @author David Graeff - Initial contribution
  */
+@Deprecated
 public interface SSLContextProvider {
```

Comment:
Using this interface is a NO-OP I guess? Deprecating something means, that it still works but will be removed in the future. If this is a NO-OP now with the custom trust manager, an interface comment "DO NOT USE. This interface does nothing." might help to clarify?

Reply:
It still works. I changed the two implementations so that this is fully transparent.